docs/install.md

```diff
@@ -130,7 +130,9 @@ You can run the example from the *artery-build* directory located in the box's h
 invoking `make run_example`.
 Please note that the *artery* directory mirrors the cloned repository on your host machine.
 
-![Starting Artery in virtual machine supplied by Vagrant](../assets/vagrant.webm){:autoplay loop}
```

Comment:
I have noticed it but I could not get a working version of vidifiy.

> I think mixing two mark-up languages in a single file is odd.

I agree except I think of Markdown more as a preprocessing language for html. So you just take away some work from the compile :D.
Also I think it is beneficial to have as little as possible dependencies.

Reply:
Maybe you can add vidifiy to the [dependencies](https://github.com/riebl/artery/pull/301/files#diff-001ee8fd3c7ce6043e2474c96960d83c5a0eff948e799f27763e996b283c4dc6R9)?

Reply:
*pip install markdown-vidify* did the trick on my system. Can you please elaborate on the problems you are facing with it?

I am more often reading the raw markdown files than the generated website. That's probably why I want the markdown to stay tidy :-)

Reply:
> Can you please elaborate

Back than I tried to install it with `apt` or `pip install vidify` did not work.
I have to say that I usually kept away from the python ecosystem.

> promising alternative

I think it would be wrong to use such plugins but I really do not care.
From my experience I think usually the more third party code you got the harder to maintain and the easier to penetrate the application is.
From my perspective if I see something like this `![type:video](video.mp4)` I would be confused.
If I see some unknown HTML-Element I know where I can look it up.
I see Markdown as a shortform of HTML.
I do not think that is really that bad.

I will check if I can simply revert that commit.

Reply:
Vidify works for me now and I reverted that commit.